test/testing/test_pytest_plugin.py

```diff
@@ -24,6 +25,34 @@ def test_rabbit_manager(rabbit_manager):
     assert "/" in [vhost['name'] for vhost in rabbit_manager.get_all_vhosts()]
 
 
+def test_rabbit_config_random_vhost(testdir):
+
```

Comment:
Sometimes I do. If the next section is a reasonably big block or particularly distinct I'll usually give it some breathing space.